index.js

```diff
@@ -889,7 +839,8 @@ module.exports = function({parent}) {
                 }
             });
             if ($meta.saveAs) {
-                let fileDir = path.dirname($meta.saveAs);
+                var filename = typeof $meta.saveAs === 'string' ? $meta.saveAs : $meta.saveAs.filename;
+                let fileDir = path.dirname(filename);
```

Comment:
We need to make sure that arbitrary paths cannot be used, due to security concerns and also isolation. This means saveAs should be a path relative to the working dir of the module (even better a subfolder), which is `path.join(this.bus.config.workDir, 'ut-port-sql', 'export')`

Here you can see how similar checks are done:
https://github.com/softwaregroup-bg/ut-port-file/blob/master/index.js#L62-L69

Reply:
Okay I will made the required changes and will update here.

Reply:
@kalinkrustev Can we keep the filename as simple name with ext, not as path ? ex. filename.csv. Becuase If the path is relative path ex(folder1/folder2/..../filename.ext), we need to handle to write dir(folder) by depthly. 

If it is simple filename, we just write 2 directory If not exists and next start to write the file stream.

Reply:
I did not understand what is the issue with relative path. You can use fs-plus to create nested directories. Let's not limit the functionality. I think we may need to use sub-folders at some point.

Reply:
I have made the required changes. please check it